Python/Product/VSInterpreters/Interpreter/CondaEnvironmentFactoryProvider.cs

```diff
@@ -285,25 +285,18 @@ class CondaInfoResult {
             public string[] EnvironmentRootFolders = null;
         }
 
-        private List<PythonInterpreterInformation> FindCondaEnvironments(string condaPath) {
-            var found = new List<PythonInterpreterInformation>();
-            var watchFolders = new HashSet<string>();
-
+        private IEnumerable<PythonInterpreterInformation> FindCondaEnvironments(string condaPath) {
```

Comment:
I see no reason not to keep returning a List here (or `IReadOnlyList`, if you prefer).